src/control/cmd/dmg/system.go

```diff
@@ -185,7 +186,8 @@ func (cmd *systemStopCmd) Execute(_ []string) (errOut error) {
 	if err := cmd.validateHostsRanks(); err != nil {
 		return err
 	}
-	req := &control.SystemStopReq{Force: cmd.Force}
+	// DAOS-16312: Always use force when stopping ranks.
+	req := &control.SystemStopReq{Force: true}
```

Comment:
This is different than the PR for master. Was that intentional? Changing it here means that the behavior will only be modified for dmg users.

Reply:
not intentional let me fix